### PR TITLE
cksum: Read file in chunks + display digest as hex

### PR DESCRIPTION
**Pull request by luk1337**

### CODE REVIEW

File: Userland/Utilities/cksum.cpp [hunk 1]
```diff
@@ -47,12 +47,22 @@ int main(int argc, char** argv)
...
+        if (fstat(file->fd(), &st) < 0) {
+            perror("fstat");
+            fail = true;
+            continue;
+        }
         if (algorithm == "crc32") {
-            outln("{} {} {}", Crypto::Checksum::CRC32 { file_buffer.bytes() }.digest(), bytes, path);
+            Crypto::Checksum::CRC32 crc32;
+            while (!file->eof() && !file->has_error())
+                crc32.update(file->read(PAGE_SIZE));
```
Comment (gunnarbeutner) [inline Userland/Utilities/cksum.cpp:59]: I know it didn't have it before, but would some error handling make sense here?

Comment (luk1337) [reply Userland/Utilities/cksum.cpp:59]: Not sure, I just copy pasted it from `checksum` program.

Comment (gunnarbeutner) [reply Userland/Utilities/cksum.cpp:59]: Well, if the read fails for some reason (e.g. an I/O error) we'd just silently output a hash that was calculated for parts of the file. How about we output an error message in that case? :)

Comment (luk1337) [reply Userland/Utilities/cksum.cpp:60]: Something like this? Also changed `perror()` to `warnln() ` to match other errors.

Comment (gunnarbeutner) [reply Userland/Utilities/cksum.cpp:60]: That's so much better. :partying_face:

**Pull request merged**